src/services/dumpers/templates/agent-nodejs/index.hbs

```diff
@@ -0,0 +1,68 @@
+require('dotenv').config();
+const { createAgent } = require('@forestadmin/agent');
+{{datasourceImport}}
+{{#unless isMongoose}}
+
+const dialectOptions = {};
+
+{{#if (or isMySQL isMariaDB)}}
+dialectOptions.typeCast = (field, useDefaultTypeCasting) => {
+  if (field.type === 'BIT' && field.length === 1) {
+    const bytes = field.buffer();
+    return bytes ? bytes[0] === 1 : bytes;
+  }
+
+  return useDefaultTypeCasting();
+};
+
+{{/if}}
+if (process.env.DATABASE_SSL && JSON.parse(process.env.DATABASE_SSL.toLowerCase())) {
+  // Set to false to bypass SSL certificate verification (useful for self-signed certificates).
+  const rejectUnauthorized =
+    process.env.DATABASE_REJECT_UNAUTHORIZED &&
+    JSON.parse(process.env.DATABASE_REJECT_UNAUTHORIZED.toLowerCase());
+{{#if isMySQL}}
+  dialectOptions.ssl = { rejectUnauthorized };
+{{else if isMSSQL}}
+  dialectOptions.options = {
+    encrypt: true,
+    trustServerCertificate: !rejectUnauthorized,
+  };
+{{else if isMariaDB}}
+  dialectOptions.ssl = !rejectUnauthorized ? { rejectUnauthorized } : true;
+{{else}}
+  dialectOptions.ssl = !rejectUnauthorized
+    ? {
+        require: true,
+        rejectUnauthorized,
+      }
+    : true;
+{{/if}}
+}
+{{/unless}}
+
+// Create the Forest Admin agent.
+/**
+ * @type {import('@forestadmin/agent').Agent<import('./typings').Schema>}
+ */
+const agent = createAgent({
+  authSecret: process.env.FOREST_AUTH_SECRET,
+  envSecret: process.env.FOREST_ENV_SECRET,
+  isProduction: process.env.NODE_ENV === 'production',
+  {{#if forestServerUrl}}
+  forestServerUrl: process.env.FOREST_SERVER_URL,
+  {{/if}}
+  // Autocompletion of collection names and fields
+  typingsPath: './typings.ts',
+})
+  // Connect your datasources.
+  .addDataSource({{datasourceCreation}});
+
+// Add customizations here.
+// agent.addCustomization('collectionName', collection => ...);
+
+agent
+  // Expose an HTTP endpoint.
+  .mountOnStandaloneServer(process.env.APPLICATION_PORT)
```

Comment:
I prefer to have the two options in the template then. Because when you generate your project localy you will never have the `PORT` env variable. The only way is to handle it right here.

Reply:
On the other hand, I don't think we should aim to specificaly support heroku or another cloud provider.
The generated files should be used to run on a local environment, and if needed the developers should modify it as they need.
For exemple if AWS has a specific default env variable for the port, it wouldn't make sense to not add it here too if the Heroku one is 🤷 

Since `forest-express` handles it, I'll add it, but in my opinion it's not a good thing.

Reply:
👍